### PR TITLE
Tutorial page

### PR DESCRIPTION
**Pull request by sarah-jensen**
You'll need to run `npm i` in the client folder in order to install dependencies to render some Material-UI components.

Click the icon next to "Lessons" to see the list of lesson titles. They are links, but the individual lesson view page hasn't been created yet, so they navigate to a blank page.

There is one console warning that I wasn't able to resolve related to having unique "key" props for .map functions. From what I can tell, the id is assigned to each item in the array as the key, and there doesn't seem to be an issue rendering the data in the browser at this time.

The Lesson and Reviews cards should change from displaying in a row to a column on smaller screens.

### CODE REVIEW

File: client/src/App.js [hunk 1]
```diff
@@ -45,6 +46,10 @@ function App() {
               path='/signin'
               element={<SignIn />}
             ></Route>
+            <Route
+            path='/tutorial/:ID'
```
Comment (sendusyourbones) [inline client/src/App.js]: Indenting needed here

File: client/src/pages/ViewTutorial.js [hunk 2]
```diff
@@ -0,0 +1,257 @@
...
+        </Box>
+        <Box direction='row'>{categoryList(categories)}</Box>
+      </Container>
+      <Grid
+        container
+        className={classes.info}
+        direction='row'
+        justifyContent='space-evenly'
+        alignItems='center'
+        // spacing={20}
```
Comment (sendusyourbones) [inline client/src/pages/ViewTutorial.js]: Remove commented out code if not needed anymore?

File: client/src/utils/queries/tutorialQueries.js [hunk 3]
```diff
@@ -10,3 +10,33 @@ export const QUERY_TUTORIALS = gql`
     }
   }
 `;
+
+//get single tutorial
+export const GET_TUTORIAL = gql`
+query GetTutorial($id: ID!) {
```
Comment (sendusyourbones) [inline client/src/utils/queries/tutorialQueries.js]: Indentation needed for this line

File: package.json
Comment (sendusyourbones) [inline package.json:1]: Please confirm that these 3 additions are needed in the root `package.json` and not `client` or `server`

**Comment by sendusyourbones** [comment]
This is really amazing work, Sarah!

Strange about the unique key error message. I did a little searching and it seems like the issue might be related to the use of fragments and needing the key to be on the fragment? https://github.com/facebook/react/issues/15620

In any event, I'm not going to expect you to fix this in order for me to approve this PR--just wanted to share that information with you in case you want to have a go at it later. We've got more important things to focus on.

Another suggestion for a future PR is to add a label to the teacher's username, something like `Created by: [username]`.

File: client/src/App.js (hunk 1 above)
Comment (sarah-jensen) [reply client/src/App.js]: ran Prettier on App.js

File: client/src/pages/ViewTutorial.js (hunk 2 above)
Comment (sarah-jensen) [reply client/src/pages/ViewTutorial.js]: deleted

File: client/src/utils/queries/tutorialQueries.js (hunk 3 above)
Comment (sarah-jensen) [reply client/src/utils/queries/tutorialQueries.js]: ran Prettier on tutorialQueries.js

File: package.json
Comment (sarah-jensen) [reply package.json:1]: moved the @mui dependencies to client folder. the @babel one removes a warning when running npm run develop and should be in root

**Review by sendusyourbones — APPROVED** [review]
Awesome work!

**Pull request merged**